### PR TITLE
Support for aria-labels on link

### PR DESCRIPTION
**Pull request by patniko**
## Changelog Entry

Added support for custom attributes in the markdown renderer.

## Description

Modern screen readers require aria-label attributes to be present on links, but the current markdown-it configuration doesn't support `aria-label` attributes. Adding support for attributes will enable users to set aria-label's and custom attributes on links in the Bot configuration interface.

Appending {aria-label="Label description"} to the end of the link sets the aria-label attribute on the link when rendered.

## Specific Changes

Imported the [`markdown-it-attrs`](https://www.npmjs.com/package/markdown-it-attrs) npm package and added it to the renderer's configuration.

--

-  [X] Testing Added

### CODE REVIEW

File: packages/bundle/src/renderMarkdown.js [hunk 1]
```diff
@@ -2,11 +2,12 @@
 
 import iterator from 'markdown-it-for-inline';
 import MarkdownIt from 'markdown-it';
+import markdownItAttrs from 'markdown-it-attrs';
 import sanitizeHTML from 'sanitize-html';
 
 const SANITIZE_HTML_OPTIONS = {
   allowedAttributes: {
-    a: ['href', 'name', 'target', 'title'],
+    a: ['href', 'name', 'target', 'title', 'aria-label'],
```
Comment (compulim) [inline packages/bundle/src/renderMarkdown.js]: Nitpicking:

```suggestion
    a: ['aria-label', 'href', 'name', 'target', 'title'],
```

**Comment by compulim** [comment]
image

I am testing `4.8.1-master.31291d5`

**Comment by compulim** [comment]
The `markdown-it-attrs` package doesn't support ES5 and will break IE11 simply by loading it.

Although it do include ES5 version, it didn't expose it properly (via `package.json/browser` field).

I am fixing it by [transpiling it before consuming/loading it](https://npmjs.com/package/markdown-it-attrs-es5). Will need to take another PR before we can produce a working build.

**Comment by compulim** [comment]
#3025 will fix this.

**Comment by corinagum** [comment]
Closing as resolved.

**Pull request merged**